variables.tf

```diff
@@ -26,7 +26,8 @@ variable "amis" {
     description = "Which AMI to spawn. Defaults to the AWS ECS optimized images."
     # TODO: support other regions.
     default = {
-        us-east-1 = "ami-ddc7b6b7"
+        us-west-1 = "ami-59412f39"
```

Comment:
i don't know how to specify this array into my terraform.tfvars as a keyvalue pair